### ISSUE TITLE
How do I clear Notification history?

### ISSUE THREAD

**Issue by miversen33** [rcarriga/nvim-notify#158]
Looking through the readme, it looks like we can view the notification history, but I am not seeing a way to clear it out. Am I missing something?

**Comment by rcarriga** [comment]
There is no way to do this. Why do you want to clear the history? Just asking to understand the need :smile:

**Comment by miversen33** [comment]
Well to clear the history of course ;)

The real answer, I have lua's print module go to nvim-notify because I prefer how it handles displaying information. 

However if I print a ton of stuff quickly, I will need to be able to do back and see those, which I can of course do with looking at the history via the provided tools.

Except the history viewing tools will start you at the beginning of the history chain. If you've got a ton of notifications because you're abusing nvim-notify *nervous glances*, then finding the section of your history you care about is painful.

I could partially use telescope to find specific notification, but I really care about the chain of them. And IMO I shouldn't have to restart my editor to clear message history, especially when there is `:messages clear`.

TLDR: lots of notifications makes it hard to find a specific _set_ of notifications

**Comment by rcarriga** [comment]
It sounds like instead of clearing history, your issue would be better solved by the history being in the order of most recent first which I think would make sense in general anyway. What do you think?

**Comment by miversen33** [comment]
I do like that option, though I still think that clearing your notifications makes alot of sense IMO.

**Comment by nullromo** [comment]
I thought this feature would be useful too, so I made a pull request (#248) to add the appropriate command.
